webapp/src/components/modals/run_playbook_modal.tsx

```diff
@@ -0,0 +1,238 @@
+import React, {ComponentProps, useState, useEffect} from 'react';
+
+import {useIntl} from 'react-intl';
+import styled from 'styled-components';
+import {useSelector} from 'react-redux';
+import {getCurrentUser, getUser} from 'mattermost-redux/selectors/entities/users';
+
+import {Client4} from 'mattermost-redux/client';
+
+import Icon from '@mdi/react';
+import {mdiNotebookOutline, mdiAccountOutline} from '@mdi/js';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+
+import {displayUsername, getFullName} from 'mattermost-redux/utils/user_utils';
+import {getTeammateNameDisplaySetting} from 'mattermost-redux/selectors/entities/preferences';
+
+import {UserProfile} from '@mattermost/types/users';
+
+import {usePlaybook} from 'src/hooks';
+import {BaseInput} from 'src/components/assets/inputs';
+import GenericModal, {InlineLabel, Description} from 'src/components/widgets/generic_modal';
+import {createPlaybookRun} from 'src/client';
+import {navigateToPluginUrl} from 'src/browser_routing';
+
+const ID = 'playbooks_run_playbook_dialog';
+
+export const makeModalDefinition = (playbookId: string, defaultOwnerId: string, description: string, teamId: string, teamName: string) => ({
+    modalId: ID,
+    dialogType: RunPlaybookModal,
+    dialogProps: {playbookId, defaultOwnerId, description, teamId, teamName},
+});
+
+type Props = {
+    playbookId: string,
+    defaultUserId: string,
+    description: string,
+    teamId: string,
+    teamName: string
+} & Partial<ComponentProps<typeof GenericModal>>;
+
+const RunPlaybookModal = ({
+    playbookId,
+    defaultUserId,
```

Comment:
This does not destructure the value correctly, on line 31 it's passed as `defaultOwnerId`. 

```suggestion
    defaultOwnerId,
```